docs/current_docs/integrations/snippets/gitlab-kubernetes.yml

```diff
@@ -0,0 +1,17 @@
+.dagger:
+  image: alpine:latest
+  variables:
+    _EXPERIMENTAL_DAGGER_RUNNER_HOST: "unix:///var/run/dagger/buildkitd.sock"
+    DAGGER_CLOUD_TOKEN: $DAGGER_CLOUD_TOKEN
```

Comment:
This should be a secret - I am assuming it is.

Reply:
Added a suggestion for this. Please review @gerhard @chrira

Reply:
LGTM